src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1783,7 +1783,10 @@ bool CanDeleteSecretKey()
         
         public Task<(long Index, object Result)> DeleteDatabaseAsync(string db, bool hardDelete, string[] fromNodes, string raftRequestId)
         {
-            var deleteCommand = new DeleteDatabaseCommand(db, raftRequestId)
+            if (raftRequestId == RaftIdGenerator.DontCareId)
+                raftRequestId = Guid.NewGuid().ToString();
```

Comment:
why?
also,  this will make the failover not to work.. right?

Reply:
No.
First, we don't want a failover when this EP is failing.
Second, when we get 'DontCareId' as 'raftRequestId', It comes from places which doesn't wait for the remove node from database commands..
Where I am waiting fro the remove command (The Delete  Database EP), I am using the real raftRequestId of the request, so even if failover will happen, It will have the same raftRequestId.